src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.GeneratingReportServiceImpl;
+import core.basesyntax.service.impl.ProcessingFileDataServiceImpl;
+import core.basesyntax.service.impl.ReadingFileServiceImpl;
+import core.basesyntax.service.impl.WritingFileServiceImpl;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.impl.BalanceOperationHandler;
+import core.basesyntax.strategy.impl.OperationStrategyImpl;
+import core.basesyntax.strategy.impl.PurchaseOperationHandler;
+import core.basesyntax.strategy.impl.ReturnOperationHandler;
+import core.basesyntax.strategy.impl.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Application {
+    private static final String INPUT_FILE = "src/main/resources/fruits_data.csv";
+    private static final String OUTPUT_FILE = "src/main/resources/fruits_report.csv";
+
+    public static void main(String[] args) {
+        // reading file
+        ReadingFileServiceImpl readingFileService = new ReadingFileServiceImpl();
+        List<String> fileLines = readingFileService.readingDataFromFile(INPUT_FILE);
+        // translate strings into a fieldset
+        ProcessingFileDataServiceImpl processingFileDataService
+                = new ProcessingFileDataServiceImpl();
+        final List<FruitTransaction> fruitTransactions
+                = processingFileDataService.processingFileData(fileLines);
```

Comment:
```suggestion
                = processingFileDataService.parseData(fileLines);
```
Why File again? Do not forget about Abstractions. Today this data is from file, tomorrow it will be from remote server. It is not important for a parsing process where this date came from.

Reply:
Yes, my mistake also, I forgot about abstraction